### DISCUSSION TITLE
Anthropic 推出最新一代 AI 模型：Claude 3

### DISCUSSION THREAD

**Discussion by OG-Matcha** [1101-GS4719-C/Python-Community-GS4719-C#1910] [科技新知]
我今天在新聞上看到 Anthropic 在昨天推出最新一代 AI 模型：Claude 3

![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/85878531/788a125c-f47c-4eb9-8007-fb8c4cfbe418)

這次 Claude 3 總共有三種不同的版本，分別為 Opus、Sonnet、Haiku。

- Claude 3 Opus 是這次最強的 AI 模型，在高度複雜的任務上表現最佳，適合處理：任務自動化、研發、策略…等複雜任務。
- Claude 3 Sonnet 則在能力與速度上取得平衡，比其他模型更適合規模化，適合處理：數據處理、銷售、節省時間的任務。
- Claude 3 Haiku 是這次速度最快、成本最低的模型，能夠快速回應問題、模擬人類互動，適合處理：客戶互動、內容審查、節省成本的任務。

這次所有的 Claude 3 模型在分析和預測能力、內容生成、程式碼生成，以及使用多國語言上，都有明顯提升。

![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/85878531/738bc491-a85e-430d-b490-86c1ead937c8)

目前我們可以免費使用的是目前在 [Claude.ai](https://claude.ai) 中的預設模型是 Claude 3 Sonnet。

我覺得既然 Claude 3 增加了視覺能力，可以處理包括照片、圖表、圖形和技術圖解等格式，或許對沒有訂閱 GPT-4 的人來說是個不錯的替代品，至少單從數據上的顯示來看是優於 GPT-3.5，以上是我的見解。

**Comment by YeeDaniel** [comment]
[參考文章](https://www.msn.com/zh-tw/news/techandscience/%E8%B6%85%E8%B6%8A-gpt-4-claude-3-%E7%A7%92%E8%AE%80%E8%90%AC%E5%AD%97-ai-%E6%96%B0%E6%99%82%E4%BB%A3%E4%BE%86%E8%87%A8/ar-BB1jm2PU?ocid=msedgdhp&pc=U531&cvid=b7b952ae2a0b4f02b6bfb0a8aa28d97b&ei=11)

今天在以上連結也有看到類似文章，如樓主所描述的，Claude3 sonnet在數據上確實是優於GPT-3.5。另外，參考文章的觀點則是著重於Claude3 Opus上，然而，目前使用上仍是以Claude3 sonnet模型進行。對此，參考文章標題應該需要註明清楚，而非模稜兩可。

**Reply by OG-Matcha** [reply]
確實像同學說的一樣，目前免費使用版的模型是 Claude 3 Sonnet。不過有另外一點可以補充。

我有看到這篇[貼文](https://twitter.com/tolgabilge_/status/1764754012824314102?s=46&t=30MiwCUSjas_pU9EJO2vsg&fbclid=PAAaaF64pX_iKAkZpZ4UQV9mQgkydHOIUM759IzMGMaII4eE344tLhjDFqXvg)

![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/85878531/f1e9c566-e601-44ea-abdf-14f8580ede9f)

貼主說了 Claude 3 是拿 GPT-4 剛釋出時的參數做比較，但目前經過改進的 GPT-4-turbo 的測試得分是比較高的。

**Reply by chia-kai-chang** [reply]
同學我辦公室有3080，4060Ti，如果有需要可以連線進來玩

**Reply by OG-Matcha** [reply]
@benctw 謝謝教授，之後有需要用到的時候再麻煩了😆

**Reply by YeeDaniel** [reply]
@benctw 謝謝教授~

**Comment by benbebobo149** [comment]
看到Anthropic推出的Claude 3系列，我對於這些不同版本的模型都感到興奮。每個版本都有其獨特的特點，但我特別看好Claude 3 Haiku。這款模型的成本最低，速度最快，這使得它非常適合應用於邊緣運算，例如自駕車或手機等設備。這種靈活性和效能的結合將為未來帶來巨大的潛力和創新。感謝同學的分享，讓我了解到這項最新的技術發展。

**Comment by JustAHedgehog** [comment]
這邊幫平常沒有關注的同學做一下科普，分享我目前查閱，學習整理出的結論。若有任何說的不清楚或說錯的，麻煩鞭小力一點><

首先，讓我們從人工智慧（AI）的類型談起。根據運作概念來看可以粗略分為生成式人工智慧（Generative AI）和分辨式人工智慧（Discriminative AI）。

分辨式人工智慧（Discriminative AI），顧名思義，就是藉由標註標籤資料（也就是一堆可憐新鮮人一天到晚貼標籤(labeling)的東西）以供模型「分辨」物件，讓AI模型學習並辨別那些物件，就像媽媽教導小嬰兒辨別水果、車子、椅子一些物品一樣，當他看懂那些是什麼東西時，就可以請他幫忙，製作一些工作囉~
怎麼使用呢？一樣舉小嬰兒在益智玩具好了（如下圖），當他發現正方形的方塊時，就將他塞進去正方形的孔洞；發現三角形，就放到三角形的孔洞中......依此類推。
![sg-11134201-22120-e2p8un8fj0kv8f](https://github.com/NCU-GS4719-Python/Python-Community/assets/143854000/a832c555-fa11-4f8b-8518-6d9cef8b489d)
這種遇到該種東西就做相對應的動作就像我們撰寫程式時，使用if, else 判斷式的狀況，所以只要我們將這邏輯應用在AI模型上，就可以請AI模型幫我們處理一些繁瑣的事情，甚至將它結合機械手臂、家電等物件，形成了智能家電這些常常聽到，貼近我們生活的東西，再延伸就造就自動化工程、工業4.0等酷炫的名詞了。

而生成式人工智慧（Generative AI），也就是生成式 AI ，與分辨式模型就根本的差異，在於生成式AI模型巧妙地利用大量未標記的資料X，借助強大的GPU運算能力和大量資料，試圖自我生成資料以及其中隱含的信息。正是這種方法導致了通用型AI如ChatGPT的出現，改變了我們日常查閱資料、創作和研究的方式。
（備註：生成式 AI 雖然沒有必要使用有標註標籤的資料，但若將有標註標籤的資料合併使用，訓練過程中再引入強化學習（Reinforcement learning）機制，可協助引導模型快速朝向「正確」合用的方向收斂。）

好的！讓我們回到主題，談談本文提及的Anthropic公司。他是一間2021年新創的AI公司，由OpenAI（創立於2015年）的前成員所創立，專注於開發通用AI系統和語言模型，並秉持負責任的AI使用理念。而此次發表的 AI 模型，Claude 3，跟Chat GPT一樣屬於所謂的生成式AI。

根據 Anthropic 所言，Claude 3 每一次處理的單詞數量高達 15 萬個，相當於一本長篇小說的內容；Claude 2 只能處理 7.5 萬個單詞，ChatGPT 只能處理 3000 個單詞。且在性能上，Claude 3 Opus 全面超越 OpenAI 的 GPT-4 和 Google 的 Gemini Ultra 等同類產品。在對應各類知識水平的理解能力、更進階的推理能力、一定的數學計算能力、多元模式的運作功能等方面，都勝出許多。

目前Claude 3可執行高級的認知和推理，也能生成程式碼，並即時翻譯、協助練習多種語言的文法。Claude 3還可以轉錄及分析各式靜態圖像，如手寫筆記、照片和圖表等。不過，Claude 3只分析圖像，不會產生圖像。對此，Anthropic執行長阿莫迪（Daniela Amodei）回應，「因為我們發現企業對圖像的需求比較少。」

以上，關於本篇文章沒有提及到，但若具備會更好理解為什麼此事件會到轟動的這些知識希望能幫助剛接觸AI人工智慧的同學們，相關引用文獻如下，若有講解錯誤或不完全的，再麻煩各位大大小力補充，我也會盡力修正，謝謝。

文獻參考：
https://www.anthropic.com/
https://news.pts.org.tw/article/684149
https://www.storm.mg/lifestyle/5043377
https://newtalk.tw/news/view/2024-03-05/910974
https://scitechvista.nat.gov.tw/Article/C000003/detail?ID=c746ecd6-5e7d-4fc1-afe3-d91f2c06b992
